commands/🔰 Info/support.js

```diff
@@ -10,33 +11,41 @@ module.exports = {
     usage: "support",
     description: "Gives you an Invite link for the Support Server",
     run: async (client, message, args, user, text, prefix) => {
-    try{
-      message.channel.send(new MessageEmbed()
-        .setColor(ee.color)
-        .setTitle(":heart: Join here!")
-        .setFooter(ee.footertext, ee.footericon)
-        .setURL(`https://discord.gg/kUuNQwnvCF`)
-        .setDescription(`[Click here](https://discord.gg/kUuNQwnvCF)`)
-        .setImage("https://cdn.discordapp.com/attachments/802144342185738250/820385232686546945/cover.png")
+        try {
+            let embed = new Discord.MessageEmbed()
+                .setColor(ee.color)
+                .setTitle(":heart: Join here!")
+                .setFooter(ee.footertext, ee.footericon)
+                .setURL(`https://discord.gg/kUuNQwnvCF`)
+                .setDescription(`[Click here](https://discord.gg/kUuNQwnvCF) (https://discord.gg/kUuNQwnvCF)`)
```

Comment:
Discord shows what URL you are opening when clicking on it, there's no need to append it afterwards.